src/app/menu-bar/menu-bar.component.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
Do we really need it here ??? Did you know that we use <html lang="en"> only one per app, we can use it also only in iframe

Reply:
ok, it’s already removed